packages/@office-iss/react-native-win32/package.json

```diff
@@ -101,4 +101,4 @@
   "engines": {
     "node": ">= 14"
   }
-}
+}
```

Comment:
See #11001, it seems like there's a configuration in VSCode that is removing the newline. I was able to add the newline back in manually and use "Save without formatting" to fix it. You could also use notepad and add them back or disable 'format on save' temporary in VsCode settings file